shared/actions/config/index.js

```diff
@@ -390,6 +391,24 @@ const updateServerConfig = (state: TypedState) =>
     }
   })
 
+const writeLastSentXLM = (_: any, action: SetLastSentXLMPayload) =>
+  Saga.call(function*() {
```

Comment:
not certain about doing it this way, it quiets down flow at least

Reply:
just do actionToPromise. lastSentXLM is store in the store right, just grab it from there, you don't need the action at all